src/python/pants/backend/python/lint/pylint/subsystem.py

```diff
@@ -47,9 +47,8 @@ def register_options(cls, register):
             advanced=True,
             help=(
                 "An optional list of `pylint_source_plugin` target addresses. This allows you to "
-                "load custom plugins defined in source code. Run the goal `target-types "
-                "--details=pylint_source_plugin` for instructions, including how to load "
-                "third-party plugins."
+                "load custom plugins defined in source code. Run `./pants help "
+                "pylint_source_plugin` for instructions, including how to load third-party plugins."
```

Comment:
Missing opening back-tick?

Reply:
On the line before. We use implicit string concat here.